src/frontend/App/hooks/useJournalføringState.ts

```diff
@@ -106,6 +108,7 @@ export const useJournalføringState = (
     const [valgtDokumentPanel, settValgtDokumentPanel] = useState<string>(
         utledFørsteDokument(journalResponse.journalpost.dokumenter)
     );
+    const [nyAvsender, settNyAvsender] = useState<string>(''); // TODO: Denne må sendes med til backend for å bli satt
```

Comment:
Tror vi må tenke litt her - for det er to-tre scenarier:
- Ingen endring på avsender, da vil vi ikke sende inn noe
- Avsender er bruker (endring) -> sende inn et eller annet... 
- Avsender er IKKE bruker -> sende inn navn...

Reply:
Ja, men dette er vel logikk som vil bli lagt inn i metoden for `fullførJournalføringV2()`?

Send inn dersom nyAvsender ikke er tom streng og ikke er det samme før? Eventuelt kan man sende inn dette uansett og la backend utlede om man skal sette ny avsender eller ikke - basert på de samme reglene